hasUniqueChars.c

```diff
@@ -47,7 +47,7 @@ void checkInvalid(char * inputStr) {
 
     // if nextChar is invalid (31 or less or 127 as an ascii char code), then bail out
     if ((nextChar <= 31 ) || (nextChar == 127)) {
-      fprintf(stderr, "invalid character in string\n");
+      fprintf(stderr, "invalid character in string dummy\n");
       exit(EXIT_FAILURE);
```

Comment:
This is already handled by the call to checkInvalid at the start of the function